Gemfile

```diff
@@ -9,11 +9,11 @@ gem 'rack-cors'
 gem 'rake'
 gem 'thin'
 
-gem 'grape', '>=1.5.0' # Important fixes are introduced in v1.5.0 (see PR #2096 & #2098)
+gem 'grape', '>=1.5.1' # Important fixes are introduced in v1.5.1 (see PRs PR #2128, #2096 & #2098)
```

Comment:
I have checked, the fix comes from PR #2103 (https://github.com/ruby-grape/grape/pull/2103/files)